app/src/main/java/org/simple/clinic/bloodsugar/history/BloodSugarHistoryScreen.kt

```diff
@@ -165,7 +158,19 @@ class BloodSugarHistoryScreen(
     activity.startActivity(intent)
   }
 
-  override fun showBloodPressures(bloodPressureHistoryListItemDataSourceFactory: BloodSugarHistoryListItemDataSourceFactory) {
+  @SuppressLint("CheckResult")
+  override fun showBloodPressures(dataSourceFactory: BloodSugarHistoryListItemDataSourceFactory) {
+    val detaches = detaches()
+    // Initial load size hint should be a multiple of page size
+    val config = Config(
+        pageSize = 20,
+        prefetchDistance = 10,
+        initialLoadSizeHint = 40,
+        enablePlaceholders = false
+    )
```

Comment:
If these values are the same for the ones we use to paginate the BP history screen, we should consider providing the values via a config object so they can be defined in one place.

Reply:
These values are the same for history screens only, should I create a pagination config for history screen only?

Reply:
Yeah, we could do that.

Reply:
Maybe something like `measurementHistoryPaginationConfig` or something.